test/jsonapi-document-structure-resource-fields.test.js

```diff
@@ -0,0 +1,454 @@
+/* eslint-env mocha */
```

Comment:
This file has `142` linting errors.

Please run the following comment to resolve these linting errors:

- `eslint **/*.js /test --fix`

If you add more test cases to this file please make sure to run the command above to ensure you are following the linting rules expectations.